apstra/two_stage_l3_clos_configlets.go

```diff
@@ -0,0 +1,174 @@
+package apstra
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	apiUrlBlueprintConfiglets       = apiUrlBlueprintById + apiUrlPathDelim + "configlets"
+	apiUrlBlueprintConfigletsPrefix = apiUrlBlueprintConfiglets + apiUrlPathDelim
+	apiUrlBlueprintConfigletsById   = apiUrlBlueprintConfigletsPrefix + "%s"
+)
+
+type rawTwoStageL3ClosConfiglet struct {
+	Data      rawConfigletData `json:"configlet"`
+	Id        string           `json:"id"`
+	Condition string           `json:"condition"`
+	Label     string           `json:"label"`
+}
+
+type TwoStageL3ClosConfiglet struct {
+	Data      ConfigletData
+	Id        string
+	Condition string
+	Label     string
+}
+
+func (o *TwoStageL3ClosConfiglet) raw() *rawTwoStageL3ClosConfiglet {
+	rawc := rawTwoStageL3ClosConfiglet{}
+	rawc.Data = *o.Data.raw()
+	rawc.Id = o.Id
+	rawc.Condition = o.Condition
+	rawc.Label = o.Label
+	return &rawc
+}
+
+func (o *rawTwoStageL3ClosConfiglet) polish() (*TwoStageL3ClosConfiglet, error) {
+	c := TwoStageL3ClosConfiglet{}
+	c.Id = o.Id
+	c.Condition = o.Condition
+	c.Label = o.Label
+	d, err := o.Data.polish()
+	c.Data = *d
```

Comment:
if `polish()` returns an error and a `nil` configlet data, we'll crash dereferencing `d` here.